docs/whatsnew/6.0.rst

```diff
@@ -140,7 +140,7 @@ This means that the :mod:`dataclasses` machinery can be used to work with
 
     >>> from dataclasses import asdict
     >>> asdict(H0)
-    {'derived': False, 'unit': Unit("km / (Mpc s)"), 'equivalencies': [], ...
+    {'default': ..., 'derived': False, 'unit': Unit("km / (Mpc s)"), 'equivalencies': [], ...
```

Comment:
Why is the `'default'` value not written out?

Reply:
I don't want to highlight `<MISSING>` over the benefits from `dataclass`

Reply:
I'm with @eerovaher - just write it out. The `...` is more likely to lead to confusion.

Or just pick a parameter which does not have `MISSING` as its default.

Reply:
I'll do the latter!

Reply:
Done!